### ISSUE TITLE
Snapping is backwards with linked belts

### ISSUE THREAD

**Issue by raiguard** [mspielberg/factorio-miniloader#60]
![image](https://user-images.githubusercontent.com/3515394/125539608-9a3d46f6-e91e-4d8a-a475-e425fea13255.png)

If you place a miniloader adjacent to a linked belt, it always flips backwards. This happens regardless of the direction of the linked belt.

**To reproduce:**
Place a linked belt in the map editor.
Place a miniloader facing into it.
The miniloader will snap in the wrong direction.

**Comment by mspielberg** [comment]
Right, `linked-belt` is a distinct belt type now, isn't it. https://github.com/mspielberg/factorio-miniloader/blob/4df5ce68c3de60b2744ec4083f229fc736ba0477/snapping.lua#L5-L11
